modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/management_bar/controls/BulkActions.js

```diff
@@ -49,14 +50,30 @@ function BulkActions({
 		actionParameterName,
 		allItemsSelectedActive,
 		onBulkActionItemClick,
+		searchParam,
 		showBulkActionsManagementBar,
 		showBulkActionsManagementBarActions,
 		showInfoPanel,
 	} = useContext(FrontendDataSetContext);
 
+	const [filters] = useContext(ViewsContext);
```

Comment:
This is actually misnamed since the first item in this array is the `[viewsState]`. See here where it's defined https://github.com/liferay/liferay-portal/blob/a5ae22605c8a3d5f2ea82454a6a96cad8de2b2bb/modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/FrontendDataSet.tsx#L1260

I also referenced other places that use the `ViewsContext` like [here in NavBar.js](https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/management_bar/controls/NavBar.js#L28-L29).

To get the filters, we can simplify this to
```
const [{filters}] = useContext(ViewsContext);
```